dir.targets

```diff
@@ -52,15 +57,21 @@
 
     <!-- Restore build tools -->
     <Exec Command="$(NugetRestoreCommand) &quot;$(SourceDir).nuget\packages.config&quot;" StandardOutputImportance="Low" />
-    
-    <Error Condition="'$(ErrorIfBuildToolsRestoredFromIndividualProject)'=='true'"
-           Text="The build tools package was just restored and so we cannot continue the build of an individual project because targets from the build tools package were not able to be imported. Please retry the build the individual project again." />
+
+    <Error Condition="'$(ErrorIfBuildToolsRestoredFromIndividualProject)'=='true' and '$(BuildingInsideVisualStudio)'=='true'"
+      Text="The build can not be completed as the build tools package was just restored. Please reload the solution to allow targets from the build tools package to be imported." />
```

Comment:
The imports won't be there until after you reload the projects. Start with a  clean enlistment (no packages) and jump right into a solution and you'll hit this. It will build because the imports are conditionalized on exist, but VS doesn't do anything to reload/reimport.